.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -43,7 +43,6 @@ stages:
             echo '##vso[task.setvariable variable=AZURE_VM_SIZE]Standard_D4s_v3'
             echo '##vso[task.setvariable variable=CONTAINER_RUNTIME]docker'
             echo '##vso[task.setvariable variable=WINDOWS_SKU]2019'
-            echo '##vso[task.setvariable variable=VNET_RESOURCE_GROUP_NAME]vnet-2019-$(Build.BuildNumber)'
```

Comment:
Can we get away with not creating these networking resources on the fly like it's currently implemented?